pipeline/splicing/calcVarPriors.py

```diff
@@ -1572,6 +1575,7 @@ def main():
     outputData.writerow(dict((fn,fn) for fn in inputData.fieldnames))
     
     for variant in inputData:
+        variantData = csv.DictReader(open(args.variantFile, "r"), delimiter="\t")    
```

Comment:
I'm a bit confused why this would need to be called for every variant in inputData: `csv.DictReader(open(args.variantFile, "r"), delimiter="\t")`

Reply:
I'm a few branches down the line here and none of the functions that have variantData as an argument (e.g. getPriorProbProteinSNS) work unless it is called every time

Reply:
that sounds strange... is there a reason it's included in this PR? I have a feeling we can find a better way once I have some more context as to what's happening with it (I don't see any reference to the variable `variantData` after it's created).

Reply:
I can remove it from this PR. It's not necessary until I actually try to get output for more than one variant. I think there is a better way also... I'm really not sure why it doesn't work more than once